src/Linear/Vector.hs

```diff
@@ -133,8 +133,13 @@ class Functor f => Additive f where
   x ^-^ y = x ^+^ negated y
 
   -- | Linearly interpolate between two vectors.
+  --
+  -- /Since linear version 1.23, interpolation direction has been reversed; now/
+  --
+  -- prop> lerp 0 a b == a
+  -- prop> lerp 1 a b == b
```

Comment:
`cabal-docspec` [doesn't support property tests](https://github.com/phadej/cabal-extras/blob/25bf27f965fbd5722c127112c92189a645c3357a/cabal-docspec/MANUAL.md?plain=1#L595-L598), which means that the CI [skips these two tests entirely](https://github.com/ekmett/linear/actions/runs/8659936783/job/23749282166?pr=178#step:24:66):

```
[   1.02921] warning[-Wskipped-property]: in comment at 135:3 skipping lerp 0 a b == a
[   1.02968] warning[-Wskipped-property]: in comment at 135:3 skipping lerp 1 a b == b
```

Could you move these to the `linear` test suite instead?

Reply:
It looks like the test suite doesn't (currently) support property tests either. I'll just make these code fragments (since they're important for documentation). Unless you want me to add something to the test suite?

Reply:
I don't really have any experience with QuickCheck, so it'd probably be better for someone else to do it. I don't think this fix should really depend on a test being available, though.

Reply:
I've added some property tests which confirm that `lerp 0` and `lerp 1` behave as advertised. Thanks again for the PR!